### PR TITLE
Fire change event from auro-radio

### PR DESCRIPTION
**Pull request by ychae-alaska**
# Alaska Airlines Pull Request

**Fixes:** #11 

## Summary:
In a scenario where there are multiple radio buttons within a radio group, and the selection of a radio button is supposed to display different content or have a custom event, I would expect to have a `change` event listener that supports bubbling. 

The scope of this work is to add the custom `change` event listener within the scope of the component to support this external event management.

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -311,6 +311,22 @@ <h1>auro-radio</h1>
         </template>
       </demo-snippet>
 
+      <p>Group with onchange</p>
```
Comment (geoffrich) [inline demo/index.html:314]: I don't we need to add anything to the demo for this. Input and change events are already being logged in the console for every radio button on the demo page.

File: src/auro-radio.js [hunk 2]
```diff
@@ -57,6 +57,13 @@ class AuroRadio extends LitElement {
     };
   }
 
+  handleChange(event) {
```
Comment (geoffrich) [inline src/auro-radio.js:60]: Can you update the "should fire a input event with correct data" test to also check for the change event?

**Review by geoffrich — CHANGES_REQUESTED** [review]
Just a couple comments. Once you make the changes, please squash your commits into a single commit that follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) (see previous commits in the repo for examples).

**Comment by blackfalcon** [comment]
It appears that the updates were made in the Github editing tools. The preference is to fork the repo, update, and run/test locally. Version control is based on [conventional commits](https://github.com/AlaskaAirlines/auro/blob/master/src/CONTRIBUTING.md#conventional-commits). This is part of the Auro Design System Contributing Guidelines.

**Comment by ychae-alaska** [comment]
> It appears that the updates were made in the Github editing tools. The preference is to fork the repo, update, and run/test locally. Version control is based on [conventional commits](https://github.com/AlaskaAirlines/auro/blob/master/src/CONTRIBUTING.md#conventional-commits). This is part of the Auro Design System Contributing Guidelines.

Hm, I saw the forking guidelines in the "Contributions" section of the README and followed that. I'm not sure what you're seeing to think that it's edited from the browser.

**Comment by blackfalcon** [comment]
@ychae-alaska you closed this PR? Is this update no longer needed? 

> Hm, I saw the forking guidelines in the "Contributions" section of the README and followed that. I'm not sure what you're seeing to think that it's edited from the browser.

The issue I see is that the comments are not formatted correctly. All the Auro repos use [conventional commits](https://github.com/AlaskaAirlines/auro-radio/blob/master/.github/CONTRIBUTING.md#conventional-commits). Why I think that you used the Github editor is because there are pre-commit hooks that should have not allowed you to push a branch without the correct commit format. 

The update request was valid and we appreciated your PR. If needed, we can assist with cleaning up the minor issues with the PR and get this merged.

File: test/auro-radio.test.js [hunk 3]
```diff
@@ -412,8 +412,13 @@ describe('auro-radio-group', () => {
     `),
     alaskaradio = el.querySelector("auro-radio[id=alaska]"),
     alaskaradioInput = alaskaradio.shadowRoot.querySelector('input');
+    
+    var changeEventDetected;
 
     let result = false;
+    el.addEventListener('change', (event) => {
+      changeEventDetected = event.type;
```
Comment (geoffrich) [inline test/auro-radio.test.js:420]:
```suggestion
      changeEventDetected = true;
```

File: test/auro-radio.test.js [hunk 4]
```diff
@@ -422,6 +427,7 @@ describe('auro-radio-group', () => {
     alaskaradioInput.click();
 
     expect(result).to.be.true;
+    expect(changeEventDetected).to.equal("change");
```
Comment (geoffrich) [inline test/auro-radio.test.js:430]:
```suggestion
    expect(changeEventDetected).to.be.true;
```

**Review by geoffrich — CHANGES_REQUESTED** [review]
Small code style request -- `changeEventDetected` sounds like it should be a boolean. Holding the event type is redundant.

Otherwise looks great!

**Comment by ychae-alaska** [comment]
@blackfalcon - I'm not too familiar with GitHub and the conventions we may follow, so I'm sorry about that! I must have done some funky things as well as accidentally closing the PR. I'd really appreciate your help in cleaning up the PR.

**Review by geoffrich — APPROVED** [review]

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**